### PR TITLE
Added alternative solution to known-issues.md

### PR DESCRIPTION
**Pull request by thomasgeens**
Ads an alternative solution to the known issues list for an issue with the error "Couldn't transfer storage on any of the endpoints" and "Check if the source device is online - we couldn't access it."

Co-authored-by: jeroendaemers44 <jeroendaemers@telenet.be>

### CODE REVIEW

**Comment by PRMerger10** [comment]
@thomasgeens : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/storage/storage-migration-service/known-issues.md [hunk 1]
```diff
@@ -228,6 +228,28 @@ at Microsoft.StorageMigration.Proxy.Service.Transfer.TransferRequestHandler.Proc
...
+Desired Access: Read Attributes, Read Control, Synchronize, Access System Security
+Disposition: Open
+Options: Synchronous IO Non-Alert, Open For Backup
+Attributes: N
+ShareMode: Read, Write
+AllocationSize: n/a
+Impersonating: CONTOSO\ServiceAccount
+OpenResult: PRIVILEGE_NOT_HELD
+```
+The actual operation being performed needs the "Open For Backup"-privileges on the source file server. Verify that your service account is granted the necessary permissions via the following Local Security Policy on this server or using a Group Policy Object:
```
Comment (nedpyle) [inline WindowsServerDocs/storage/storage-migration-service/known-issues.md:250]: This is a great addition to the doc!

I recommend saying "source migration account", not "service account". A reader might think we're referring to the SMS orchestrator or proxy services themselves, which run as Network Service and shouldn't be touched.

**Review by nedpyle — CHANGES_REQUESTED** [review]
One change suggested, after that this should be shipped. Excellent addition to the content.

**Comment by PRMerger15** [comment]
@thomasgeens : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by thomasgeens — COMMENTED** [review]
Issue solved in additional commit #[3cbfd76](https://github.com/thomasgeens/windowsserverdocs/commit/3cbfd76d1b440c1bbb2509e9a58c7d53e114e29b)

File: WindowsServerDocs/storage/storage-migration-service/known-issues.md (hunk 1 above)
Comment (thomasgeens) [reply WindowsServerDocs/storage/storage-migration-service/known-issues.md]: Issue solved in additional commit #[3cbfd76](https://github.com/thomasgeens/windowsserverdocs/commit/3cbfd76d1b440c1bbb2509e9a58c7d53e114e29b)

**Review by nedpyle — APPROVED** [review]
Awesome edit

**Comment by thomasgeens** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@thomasgeens: I'm sorry - only the author of this article, @nedpyle, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by nedpyle** [comment]
#sign-off

**Comment by thomasgeens** [comment]
Hi @nedpyle, I'm not up to date on the release management process for Microsoft Docs. But when does a merge actually takes place?

**Pull request merged**